guides/custom_patient_page.md

```diff
@@ -0,0 +1,43 @@
+Custom patient buttons
+======================
+
+Cliniko offers an additional integration option for applications that handle
+patient data. This allows users to open these applications directly from Cliniko
+and potentially synchronize information on a given patient as needed.
+
+How it works for the user
+-------------------------
+
+Users can add these applications in a settings page under
+Settings > Our clinic > Integrations:
+
+![Configuration
+panel](assets/custom-patient-buttons-config.png?raw=true)
+
+This adds a new button to the Patient Actions bar for each individual patient:
```

Comment:
This adds a new button to the Patient Actions bar for each ~individual~ patient: